### ISSUE TITLE
Stremio stops playing

### ISSUE THREAD

**Issue by QbusKoen** [shivasiddharth/Stremio-RaspberryPi#9]
Hi there,

First of all i want to thank you for sharing this code.

But i have an issue. I can start playing a video, buffering goes fine. But after +-10s the video stops playing. Altough the video is buffered for 100%, it appears that stremio is still buffering?? When i press play, the video immadiatly starts again and then stops again after +-10.

Kind regards,

**Comment by shivasiddharth** [comment]
To be 100% sure that the video has been buffered, click or move your mouse over the globe icon. That will show the amount of video that has been buffered.

**Comment by QbusKoen** [comment]
Thx for your repply, but I checked the globe icon, there i see that the video is buffered for 100%.
I press play, 10s it goes fine, then i get buffering and it seems like it stays that way forever. When i press play, the video continues, again for about 10s.
Very strange...

**Comment by shivasiddharth** [comment]
Which board are you using ? Also, what is the video resolution that you are trying to play ?

**Comment by QbusKoen** [comment]
I first tried on a 3B+, now i'm on a 4 with 2GB. Same problem on both. For some reason i can't find the resolution settings but i'm on HDMI, i guess 1080i.
Strange thing is that video is buffered at 100% with only 512KB loaded. 
When I take a look at your server application, there i still see that it is receiving data.
I'm using TPB+ plugin by the way.

**Comment by shivasiddharth** [comment]
I meant the video resolution. Whether it is 720p, 1080p or 4K ? Can you share the screenshot of the mouse over the globe icon ?

**Comment by QbusKoen** [comment]
I'ts 1080p.
Sure, here they are:
![IMG_3166](https://user-images.githubusercontent.com/54985218/99958124-191c8100-2d89-11eb-9135-ea677cc5c589.jpg)
![IMG_3167](https://user-images.githubusercontent.com/54985218/99958148-20dc2580-2d89-11eb-8c4a-df203ae2b8fe.jpg)

**Comment by shivasiddharth** [comment]
Can you try a 720p video ? Need to see if it's an issue with dropped frames.

**Comment by QbusKoen** [comment]
Well, I did some extra tests. Sorry but I misunderstood you before. I thought you where referring to output resultion of the devide, not the video.
The video file had no refference wich resolution it has.
So I tried another file with 720p, which plays fine.
Then I tried another one with 1080p, which also plays fine.
The file i was playing before plays fine on my pc, There is a usergroup of 54 people, which all have the complete seasons - so i thought to continue with these files.
The only thing i can add to this is that i think it is a resolution of maybe 480p. It has the SciFi logo, so it must be recorded from a tv, early 2000.
Thank you so much for this fine piece of software! Now i gonna install HDMI-CEC to control Stremio with my IR remote... This is great!
